Sources/JWTKit/JWTSigners.swift

```diff
@@ -280,3 +295,231 @@ private struct JWKSigner {
         }
     }
 }
+
+/// JWTKeyCollection was introduced in v5 and replaces ``JWTSigners``.
+///
+/// - Note: Please migrate over to ``JWTKeyCollection`` before updating to v5, though if you plan on remaining on v4, ``JWTSigners`` can continue to be used.
+public actor JWTKeyCollection {
+    var signers = JWTSigners()
```

Comment:
This is a lightweight implementation of the new JWTKeyCollection API falling back entirely on the now deprecated JWTSigners API.